virtual_keyboard/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  height: inherit;
+}
+
+main {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+button {
```

Comment:
use class selectors instead tag, everywhere, except body, html and reset style tags. In future u can change tag, but class will stay the same

Reply:
Then I should give classes to all buttons. Yeap, you are right, but in this case, where I have a lot of buttons, it's okay. :/

Reply:
U can give each button the same class and it is good chance to practice BEM on this task.